AndroidStudioProject/app/src/main/java/example/watarusuzuki/android_example_using_danger/MainActivity.java

```diff
@@ -0,0 +1,54 @@
+package example.watarusuzuki.android_example_using_danger;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
+
+    private Intent mIntent;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        while (true) {
+            Intent intent = getIntent();
+            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+            startActivity(intent);
+            if (intent.hasExtra("hoge")) {
+                convertDaysToMilliseconds(1);
+                break;
+            }
+        }
+    }
+
+    private long convertDaysToMilliseconds(int days) {
+        String s = "";
+        String[] field = { hogefuga != null ? hogefuga.toString() : null };
+        for (int i = 0; i < field.length; ++i) {
+            s = s + field[i];
+        }
+
+        return 1000*3600*24*days;
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/31b5b96006304bc5f0cf6b0851acd13b8431f98d/AndroidStudioProject/app/src/main/java/example/watarusuzuki/android_example_using_danger/MainActivity.java#L35"></span>Result of integer multiplication cast to long in example.watarusuzuki.android_example_using_danger.MainActivity.convertDaysToMilliseconds(int)</td>
    </tr>
  </tbody>
</table>